### DISCUSSION TITLE
Devhouse?

### DISCUSSION THREAD

**Discussion by bukinoshita** [frontendbr/forum#62] [Dicas]
Galera, uma startup brasileira está disponibilizando um salão comercial para todos os desenvolvedores, qualquer um pode entrar e é 24/7. Totalmente Free, Wifi Free e Coffee Free. A ideia da startup é reunir desenvolvedores pra criar projetos, aprenderem e ensinarem tudo isso como uma comunidade.

Queria saber o ponto de vista e opinião de vocês e discutir sobre a ideia.

**TLDR:** http://devhouse.io/

**Comment by karlprieb** [comment]
Uma pena que isso só acontece em SP e RJ :(

Aqui no RS falta iniciativas assim.

**Comment by vitorbritto** [comment]
O DevHouse é recente @BuKinoshita ?

Eu acho fantástico poder compartilhar conhecimento e ideias. Sem contar que o _networking_ fica mais frequente. Eu, por exemplo, sou Freelancer e meu _networking_ se baseia nos fóruns, redes sociais e conferências/hangouts. 

IMHO, a melhor maneira para se aprender é com as pessoas. O motivo é simples: experiências vividas, problemas enfrentados, desafios superados. Saber como um colega / amigo lidou com determinado problema e conseguiu superá-lo em um determinado projeto é disparado a melhor solução. Sem falar que pode ser muito útil mais para frente a informação passada. Afinal, nesse mundo de Desenvolvimento acontece de tudo. =P  

Em um ambiente de trabalho colaborativo, todo dia é um novo aprendizado. =]

Aqui em Salvador havia um espaço para Coworking. Não sei como está agora.

**Comment by bukinoshita** [comment]
É bem recente @vitorbritto.

Trabalhando remoto, eu também acho bem bacana. E mesmo se trabalhasse 9-5, poder ir depois do trabalho lá pra aprender algo novo, trabalhar em um projeto pessoal ou simplesmente socializar é bem interessante. O mais bacana é que é 24/7, então pode ir qualquer hora, qualquer dia. Vai ter meetups rolando, boa oportunidade pra quem quer dar palestras, aprender ou simplesmente codar de boas.

**Comment by mksanti** [comment]
Muito foda essa ideia 😍

**Comment by karlprieb** [comment]
Valeu @BuKinoshita. Vou me inscrever :) Tava tentando lembrar de onde eu ja tinha falado contigo.... lembrei que ja conversamos no slack quando tu tava vindo para cá no BrazilJS :)

**Comment by bernardodiasc** [comment]
Faço das palavras do @vitorbritto as minhas, concordo com tudo. Aqui em Ubatuba o movimento tecnológico esta se fortalecendo, especialmente em áreas como robótica (arduino, raspberrypi, etc), no segmento de web comecei um estudio chamado Avante (http://x-team.com/2015/09/avante-studio-way-helping-others-unleash-potential/) que tinha o intuito de ser um espaço de aprendizado e trabalho. O foda eh a vida que vem e muda os planos de todos, mas individualmente ainda estamos na batalha. O local que funcionava o estúdio eh o UbaLab, então hoje em dia ao invez de dedicar esforços numa iniciativa específica estou contribuindo para o UbaLab, que vai muito além de web e tecnologias digitais. Muitas pessoas diferentes frequentam o lugar, alguns cientistas incríveis com ideias e conhecimentos que nos fazem sair completamente da caixa, é bem legal. O UbaLab eh, além de outras coisas, um espaço de coworking do tipo contribui quem pode com o quanto (ou o que) pode.

Parabéns pelo DevHouse, é uma ideia excelente e tem muito potencial!

**Comment by felipefialho** [comment]
Legal pra caramba @BuKinoshita, to pensando em me inscrever. Já participou?

**Comment by bukinoshita** [comment]
Não @LFeh. Estão em beta, então teve nenhum ainda... tão validando a ideia e coletando usuários, então se inscrever irá ajudar.

**Comment by tcelestino** [comment]
Ideia sensacional. Já no aguardo do convite.

**Comment by douglasmonteiro-dev** [comment]
E ae galera como anda esse tipo de ideia em 2019? ainda existe a devhouse? o domínio pelo menos ja está off

**Comment by brunooomelo** [comment]
tava lendo pensando que era novo kkk, depois que eu vi `2016`

**Comment by douglasmonteiro-dev** [comment]
Eu acho que esse é o maior problema de iniciativas desse tipo, no começo são muito quentes e todo mundo adora, mas infelizmente não conseguem se manter por muito tempo :/

**Comment by brunooomelo** [comment]
creio que se fosse algo online, poderia ter mais gente.
